test/e2e/specs/__snapshots__/templates.test.js.snap

```diff
@@ -10,7 +10,7 @@ exports[`templates Using a CPT with a predefined template Should add a custom po
 <!-- /wp:paragraph -->
 
 <!-- wp:quote -->
-<blockquote class=\\"wp-block-quote\\"><p></p></blockquote>
+<blockquote class=\\"wp-block-quote\\"></blockquote>
```

Comment:
Do we know why it no longer prints `<p />`?

Reply:
Because the initial content of quote will be undefined. This is how it was before the rich text value merge:

https://github.com/WordPress/gutenberg/pull/7890/files#diff-632d636fb9c39f4f593363386f77cedf